lambda/src/main/scala/pricemigrationengine/services/Zuora.scala

```diff
@@ -21,15 +21,11 @@ trait Zuora {
       payload: ZuoraAmendmentOrderPayload
   ): ZIO[Any, ZuoraOrderFailure, Unit]
 
-  def applyAmendmentOrder_json_values(
-      subscription: ZuoraSubscription,
-      payload: Value
-  ): ZIO[Any, ZuoraOrderFailure, Unit]
-
-  def renewSubscription(
+  def applyOrderAsynchronously(
```

Comment:
Perhaps some scaladoc to explain what it's for and when to use it.

Reply:
Not exactly scaladoc, but added comments, notably to explain the use of `operationDescriptionForLogging`.